cogs/reddit.py

```diff
@@ -0,0 +1,441 @@
+from discord import Embed, Color
+from discord.ext.commands import Cog, Bot, command, Context, is_nsfw
+import praw
+import os
+import random
+from .utils import constants
+
+reddit = praw.Reddit(
+    client_id=os.getenv("REDDIT_CLIENT_ID"),
+    client_secret=os.getenv("REDDIT_CLIENT_SECRET"),
+    user_agent=os.getenv("REDDIT_USER_AGENT"),
+    username=os.getenv("REDDIT_USERNAME")
+)
+
+
+class Reddit(Cog):
+    def __init__(self, bot: Bot) -> None:
+        self.bot = bot
+
```

Comment:
I'd like to see a `>>reddit` command and all of the other ones here would only act as it's subcommand to avoid too many commands in help as these are basically only different subreddits anyway.

I'd suggest making `>>reddit memes`, etc. and when the main command gets run (`reddit`) as a group itself, it will either print help or print random reddit post from given subreddit